source/stylesheets/main.scss

```diff
@@ -382,3 +382,7 @@ select:hover, select:focus, .mak:hover, .mak:focus {
 .vertical-scroll {
 	overflow-y: auto;
 }
+
+[ng-click] {
+	cursor: pointer;
+}
```

Comment:
This might result in disabled buttons which have `ng-click` defined: they could get pointer cursor, incorrectly.

Define pointer cursor for the element you're aiming for specifically, in its stylesheet (`.icon.icon-danger`, though it would be better if it would also receive a class indicating its functionality).

Reply:
good point, will change it